toolbox/hgfs/archive.go

```diff
@@ -229,42 +229,37 @@ func archiveRead(u *url.URL, tr *tar.Reader) error {
 
 // archiveWrite writes the contents of the given source directory to the given tar.Writer.
 func archiveWrite(u *url.URL, tw *tar.Writer) error {
-	dir := filepath.Dir(u.Path)
 
 	return filepath.Walk(u.Path, func(file string, fi os.FileInfo, err error) error {
 		if err != nil {
-			return filepath.SkipDir
+			return err
```

Comment:
This was intentional, behaves like the tar command (tho tar prints a warning in this case).